source/coffee/image-picker.coffee

```diff
@@ -26,14 +26,11 @@ sanitized_options = (opts) ->
   jQuery.extend(default_options, opts)
 
 both_array_are_equal = (a,b) ->
-  if (!a || !b) || (a.length != b.length)
-    return false
-  b = b[..]
-  for x in a
-    i = b.indexOf x
-    if i < 0
-      return false
-    b.splice(i, 1)
+  return false if (!a || !b) || (a.length != b.length)
```

Comment:
I think that sorting the arrays and comparing each element sequentially will be faster than using `indexOf`, but I have not done any testing to validate my hypothesis. For small data sets the performance difference will almost certainly not be noticeable.